app/js/ContentCacheManager.js

```diff
@@ -41,15 +46,17 @@ class PdfStreamsExtractor {
     this.inStream = false
     this.streamStartIndex = 0
     this.buffers = []
+    this.lastChunk = Buffer.alloc(0)
   }
 
   consume(chunk) {
     let chunkIndex = 0
     const pdfStreams = []
+    chunk = Buffer.concat([this.lastChunk, chunk])
```

Comment:
If we need more efficiency we can do this without needing to concat the buffers, by handling the overlap as a separate case.  Let's leave that for later though.